lta/bundler.py

```diff
@@ -233,6 +235,21 @@ def __init__(self, config: Dict[str, str], logger: Logger) -> None:
         }
         await lta_rc.request('POST', '/Files/actions/bulk_update', update_body)
 
+
+def check_drain_semaphore() -> bool:
+    """Check if a drain semaphore exists in the current working directory."""
+    cwd = os.getcwd()
+    drain_filename = os.path.join(cwd, DRAIN_SEMAPHORE_FILENAME)
+    return Path(drain_filename).exists()
+
+
+def check_stop_semaphore() -> bool:
+    """Check if a stop semaphore exists in the current working directory."""
+    cwd = os.getcwd()
+    drain_filename = os.path.join(cwd, STOP_SEMAPHORE_FILENAME)
+    return Path(drain_filename).exists()
```

Comment:
As I point out later, this function seems unnecessary.

Reply:
Yep, this one went away.